tool/tbot/botfs/botfs.go

```diff
@@ -76,6 +76,10 @@ const (
 	// Directories need the execute bit set for most operations on their
 	// contents to succeed.
 	DefaultDirMode fs.FileMode = 0700
+
+	// OpenMode is the mode with which files should be opened for reading and
+	// writing.
+	OpenMode int = os.O_CREATE | os.O_RDWR
```

Comment:
```suggestion
	OpenMode os.FileMode = os.O_CREATE | os.O_RDWR
```

Reply:
On second thought, this change results in a frankly bonkers amount of casting since exactly zero of Go's filesystem APIs can agree on what type file modes should be. I think I'd rather keep this as an int.